Formula/llvm.rb

```diff
@@ -254,8 +321,21 @@ def caveats
           return 0;
         }
       EOS
-      system "#{bin}/clang++", "test.cpp", "-o", "test"
-      system "./test"
+
+      # test xcode
+      # test clt
+      # set libcxx
```

Comment:
These are placeholders for (more advanced) tests, not comments. not sure about libcxx part though.